android/src/main/java/com/svtek/reactnative/RNAndroidSharerModule.java

```diff
@@ -62,15 +62,45 @@ private String getMimeType(String filePath) {
     }
   }
 
+  private boolean checkAppInstall(String uri) {
+    PackageManager pm = this.reactContext.getPackageManager();
+    try {
+        pm.getApplicationInfo(uri, 0);
+        return true;
+    } catch (PackageManager.NameNotFoundException e) {
+        // Error
+    }
+    return false;
+  }
+
+  private void shareContentViaFacebook(String filePath, String shareUrl, final Promise promise) {
+    if (!shareUrl.isEmpty()) {
+      ShareDialog shareDialog = new ShareDialog(this.reactContext.getCurrentActivity());
+      ShareLinkContent content = new ShareLinkContent.Builder().setContentUrl(Uri.parse(shareUrl)).build();
+      shareDialog.show(content, ShareDialog.Mode.WEB);
+    } else {
+      String mimeType = this.getMimeType(filePath);
+      if (mimeType.startsWith("image")) {
+          this.shareImageViaFacebook(filePath, promise);
+      } else {
+          this.shareVideoViaFacebook(filePath, promise);
+      }
```

Comment:
Please use two spaces as tabs.

Reply:
All tab inconsistencies fixed by Android Studio Reformat Code.

Reply:
I see, but there are tab inconsistencies in the file at the moment. You can check and change this in Android Studio editor preferences. Some if and try catch blocks are indented 2 spaces, some are indented 4 spaces in this commit's changes. Please make them all 2 spaces by hand if Android Studio makes it hard to do.

![Screen Shot 2020-01-15 at 23 45 27](https://user-images.githubusercontent.com/435000/72469926-3db2de80-37f1-11ea-817b-1d11ca2aae79.png)